Parent/Parent/Profile/ProfileViewController.swift

```diff
@@ -0,0 +1,161 @@
+//
+// Copyright (C) 2019-present Instructure, Inc.
+//
+// This program is free software: you can redistribute it and/or modify
+// it under the terms of the GNU General Public License as published by
+// the Free Software Foundation, version 3 of the License.
+//
+// This program is distributed in the hope that it will be useful,
+// but WITHOUT ANY WARRANTY; without even the implied warranty of
+// MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
+// GNU General Public License for more details.
+//
+// You should have received a copy of the GNU General Public License
+// along with this program.  If not, see <http://www.gnu.org/licenses/>.
+//
+
+import UIKit
+import CanvasKeymaster
+import Core
+
+private let drawerTransitioningDelegate = DrawerTransitioningDelegate()
+
+class ProfileViewController: UIViewController {
+
+    enum Cells: String, CaseIterable {
+        case observees
+        case help
+        case changeUser
+        case logOut
+        case developerMenu
+
+        func localized() -> String {
+            switch self {
+            case .observees: return NSLocalizedString("Manage Children", bundle: .parent, comment: "")
+            case .help: return NSLocalizedString("Help", bundle: .parent, comment: "")
+            case .changeUser: return NSLocalizedString("Change User", bundle: .parent, comment: "")
+            case .logOut: return NSLocalizedString("Log Out", bundle: .parent, comment: "")
+            case .developerMenu: return NSLocalizedString("Developer Menu", bundle: .parent, comment: "")
+            }
+        }
+    }
+
+    @IBOutlet weak var avatarImageView: UIImageView!
+    @IBOutlet weak var tableView: UITableView!
+    @IBOutlet weak var name: UILabel!
+    @IBOutlet weak var email: UILabel!
+
+    #if DEBUG
+    var showDevMenu = true
+    #else
+    var showDevMenu = UserDefaults.standard.bool(forKey: "showDevMenu")
+    #endif
+
+    public static func create() -> ProfileViewController {
+        let controller = self.loadFromXib()
+        controller.modalPresentationStyle = .custom
+        controller.transitioningDelegate = drawerTransitioningDelegate
+        return controller
+    }
+
+    public override func viewDidLoad() {
+        super.viewDidLoad()
+
+        let user = CanvasKeymaster.the().currentClient?.authSession.user
```

Comment:
Can we pull from AppEnvironment.shared instead?

Reply:
Currently `AppEnvironment` doesn't keep a reference to the currently logged in user. Do you think we should or were you thinking something else?

Reply:
oh yeah, it's `Keychain.currentSession` I kinda wish it was on `AppEnvironment` though. we can `userDidLogin`, and it could hold onto the entry that's given.

Reply:
Ah yeah ok I can do that.